signbank/dictionary/templates/dictionary/gloss_detail_advanced.html

```diff
@@ -345,5 +346,34 @@ <h2 id='modalTitle'>
             </table>
         </div>
     </div>
+
+    <div>
+        <hr style="border-top-color:#C0C0C0;margin-bottom:10px;">
+
+        {% if user.is_authenticated %}
```

Comment:
Is there a situation where this template is rendered when the user is _not_ authenticated?

Reply:
Good question! I don't know, sorry.

Reply:
@jonholdsworth can you please check, and remove the if condition if this template is not rendered by anything that doesn't require user authentication.

Reply:
Ok -
Code in adminviews.py checks if the user has permission to view the Dataset the gloss belongs to. This is in dispatch() so presumably it prevents access to the rest of the page on a fail.
Other code checks for authentication for showing history data.
But there is nothing specific in the code for checking Comment authentication that I can see. Comments are also handled by the Django Comments middleware, so that leans on the Auth middleware.
Short answer - I can't see how they'd get to this page without being authenticated, and if they're authenticated then they should be able to make comments.
I will remove the condition from the template.